test/test_core.py

```diff
@@ -406,11 +414,6 @@ def get_bullet_library(self, use_cmake):
                             configure_args=configure_args,
                             cache_name_extra=configure_commands[0])
 
-  @also_with_standalone_wasm()
-  @also_with_wasmfs
-  def test_hello_world(self):
-    self.do_core_test('test_hello_world.c')
```

Comment:
I kind of like having this tests as the most basic simple test in the suite.  Its the one I normally run before anything else.

Reply:
Ok, reverted.